### PR TITLE
example 6 doc: update README and scripts that contain flux commands

### PR DESCRIPTION
**Pull request by cmoussa1**
This PR makes the following changes:

1. **README.md**
- Fix format of instructions
- Remove `FLUX_SCHED_OPTIONS` instruction from walkthrough
2. **ensemble.sh**
- Change Flux commands to line up with the new execution system
- Decrease the number of jobs launched per ensemble to 750
3. **parent.sh**
- Change Flux commands to line up with the new execution system
- Decrease the number of cores from 216 to 3
_(note: This change is noted in the README, informing the user that they can increase both the number of cores in the respective scripts)_
4. `rm kvs-watch-until.lua`
- since this file is no longer used by either script, it is removed from this directory

### CODE REVIEW

File: example6/README.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
-### Hierarchical Launching
+### Example 6 - Hierarchical Launching
 
-- Launching over one million sleep 0 task
+#### Description: Launch a very large ensemble of sleep 0 tasks
```
Comment (dongahn) [inline example6/README.md]: I would rephrase this as "Launch an ensemble of ..." since we are using smallish sizes for a demo.

File: example6/README.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
-### Hierarchical Launching
+### Example 6 - Hierarchical Launching
```
Comment (dongahn) [inline example6/README.md]: Very minor nit: Use present tense for commit msgs?

```
Fix format of instructions and remove
FLUX_SCHED_OPTIONS instruction from walkthrough
``

File: example6/ensemble.sh [hunk 3]
```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env sh
```
Comment (dongahn) [inline example6/ensemble.sh]: Nit: In the commit message: Changed -> Change.

Comment (cmoussa1) [reply example6/ensemble.sh]: Ah, I keep making this mistake! Force of habit of mine to use past tense. My apologies - changes have been made and re-pushed.

**Comment by dongahn** [comment]
LGTM!

**Comment by cmoussa1** [comment]
Thanks!
